CloneTools/src/db_clone.erl

```diff
@@ -159,25 +159,29 @@ clone_account_db(Db) ->
             ]
     end.
 
-clone_all_docs(Db) ->
-    case find_missing_ids(Db) of
-        [] -> ?LOG_GREEN("  documents are in sync~n", []);
-        Ids ->
-            ?LOG_CYAN("  found ~p missing documents~n"
-                      ,[length(Ids)]),
-            clone_docs(Ids, Db, 'true')
-    end.
-
-find_missing_ids(Db) ->
-    Source = source_request([Db
-                             ,<<"_all_docs">>
-                            ]),
-    Target = target_request([Db
-                             ,<<"_all_docs">>
-                            ]),
-    SourceIds = sets:from_list(get_ids(Source)),
-    TargetIds = sets:from_list(get_ids(Target)),
-    sets:to_list(sets:subtract(SourceIds, TargetIds)).
+clone_all_docs(Db, Skip, Limit) ->
+  case find_missing_ids(Db, Skip, Limit) of
+    [] -> case Skip =:= 0 of
```

Comment:
We typically try to avoid too much nesting. I think you can accomplish this with a simple guard on the top-level case clauses:

```erlang
case find_missing_ids(...) of
    [] when Skip =:= 0 -> ?LOG_GREEN(...);
    [] -> 'ok';
    Ids -> ...
end.
```

Also, project standard is to use four spaces for indentation (following the emacs erlang-mode standards).